tests/test_cli.py

```diff
@@ -678,6 +678,37 @@ def test_run_non_universal_newline(self):
         self.assertEqual(
             (ctx.returncode, ctx.stdout, ctx.stderr), (1, expected_out, ''))
 
+    def test_run_list_files(self):
+        self.maxDiff = None
+
+        with RunContext(self) as ctx:
```

Comment:
> You can take inspiration in test_find_files_recursively() scenarios to create a good one, I would say at least (1) check the output without config, and (2) check output with a smart combination of yaml-files and ignore config options. Please think about trickly cases, if any!

I've looked at the existing test cases but hesitated to replicate all (edge) cases for `list-files`.

I went with checking without config (`(1)`) and a "smart" combination of `ignore:` and `yaml-files:`. One could come up with more complex examples but I feel we would overtest things covered elsewhere.

Reply:
:+1: